### ISSUE TITLE
Labels for LibriSpheech

### ISSUE THREAD

**Issue by MordehayM** [filippogiruzzi/voice_activity_detection#29]
Hi,
you wrote: "I automatically annotated the test-clean set of the dataset with a pretrained VAD model", my question is - 
what pretrained-model did you use to get the labels of the LibriSpeech dataset?

Thank you!

**Comment by dhirajnitk** [comment]
I am also interested in getting the labeled dataset.
Can the author share the training dataset labels
